Example/Tests/CISAdAnalyticsTestDouble.swift

```diff
@@ -124,9 +124,8 @@ class CISAdAnalyticsTestDouble: NSObject, CISAdAnalyticsProtocol, TestDoubleData
 
     func reportPlaybackMetric(_ key: String, value: Any?) {
         spy(functionName: "reportPlaybackMetric", args: [
-               "key": key,
-               "value": "\(value ?? "")"
-           ])
+            key: "\(value ?? "")"
+        ])
```

Comment:
This should have never worked previously. However, due to the issue described in the description, this went unnoticed.